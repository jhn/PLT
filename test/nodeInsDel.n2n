actor: Node = { name: String, age: Int };

fn main() -> Void {
	Keanu: Node = actor["Keanu", 35];
	print(Keanu);
	print("\n");

	Keanu [+] visited: Bool;
	Keanu.visited = true;
	print(Keanu);
	print("\n")

	Keanu [-] visited: Bool;
	print(Keanu);
	print("\n");

}
