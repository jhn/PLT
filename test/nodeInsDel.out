actor{ name = Keanu, age = 35 }
actor{ name = Keanu, age = 35, visited = true }
actor{ name = Keanu, age = 35 }
